core/src/test/scala/unit/kafka/tools/DumpLogSegmentsTest.scala

```diff
@@ -117,14 +93,10 @@ class DumpLogSegmentsTest {
     log.flush(false)
   }
 
-  @AfterEach
-  def tearDown(): Unit = {
-    Utils.closeQuietly(log, "UnifiedLog")
-    Utils.delete(tmpDir)
```

Comment:
Where do we delete tmpDir?

Reply:
TestUtils creates every file, including dirs, using deleteOnExit, so we don't need to explicitly delete them.

Reply:
Fair enough.